src/tests/all.test.ts

```diff
@@ -198,7 +209,36 @@ describe('PouchCollection Instance', () => {
                 expect(newguys).toHaveLength(0);
             });
 
-        })
+        });
+        describe('upsert with', () => {
```

Comment:
Great tests! 
But could you also add some quick ones under "with data" to cover finding items from a PouchCollection created by class models?

Just `find` should be fine.

Reply:
Can do!

Reply:
This should be fixed.